index.html

```diff
@@ -9,42 +9,55 @@
 	<link rel="stylesheet" href="style.css">
 	<title>Portfolio</title>
 </head>
-
 <body>
 	<main class="container">
-		<div class="first-section">
-			<header class="notification">
-				<nav class="nav-class">
-					<ul class="mobile-view">
-						<li class="flex-logo" href="https://"><img class="logo" src="img/global_minbar.png"
-								alt="Logo"></a>
-						<li class="hamburger"><img src="img/Union.png" alt="menu">
-				</nav>
-				</ul>
-				<div class="view-b">
-					<ul class="desktop-view">
-						<li class="not1 not">Portfolio</li>
-						<li class="not2 not">About</li>
-						<li class="not3 not">Contact</li>
-					</ul>
-				</div>
-				</nav>
-			</header>
-			<section class="main_body">
-				<h1 class="header2">I'm Ali<br>Glad to see you!</h1>
-				<p class="body2">I’m a software developer! I can help you build a product , feature or website Look
-					through some of my work and experience! If you like what you see and have a project you need coded,
-					don’t hestiate to contact me.</p>
-				<p class="connect">LET'S CONNECT</p>
-				<ul class="icons">
-					<li class="twitter"><a href="https://"><img src="img/Icon - Twitter.png" alt="Twitter"></a></li>
-					<li class="linkedin"><a href="https://"><img src="img/Icon - Linkedin.png" alt="Linkedin"></a></li>
-					<li class="medium"><a href="https://"><img src="img/Icon - Medium.png" alt="Medium"></a></li>
-					<li class="github"><a href="https://"><img src="img/Icon -GitHub.png" alt="GitHub"></a></li>
-					<li class="angelist"><a href="https://"><img src="img/Icon - Angelist.png" alt="Angelist"></a></li>
-				</ul>
+		<div class="menu-background">
+			<div class="menu-close">
+				<i class="fa-solid fa-xmark fa-2x"></i>
+			</div>
+			<ul id="menu-items">
+				<li class="menu-portfolio">
+					<a class="side-menu" href="#portfolio-start">Portfolio</a>
+				</li>
+				<li class="menu-about">
+					<a class="side-menu" href="#about">About</a>
+				</li>
+				<li class="menu-contact">
+					<a class="side-menu" href="#contact">Contact</a>
+			    </li>
+			</ul>
 		</div>
-		<section class="all_projects">
+			<div class="first-section">
+				<header class="notification">
+					<nav class="nav-class">
+						<ul class="mobile-view">
+							<li class="flex-logo" href="https://"><img class="logo" src="img/global_minbar.png" alt="Logo"></a>
+							<li class="hamburger"><img id="menu" src="img/Union.png" alt="menu">
+						</ul>
+						<div class="view-b">
+							<ul class="desktop-view">
+								<li class="not1 not">Portfolio</li>
+								<li class="not2 not">About</li>
+								<li class="not3 not">Contact</li>
+							</ul>
+						</div>
+					</nav>
+				</header>
+				<section class="main_body">
```

Comment:
- [optional] Please kindly format these lines of code properly in accordance to HTML/CSS best [practises](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). 👍 **Kindly go through your codebase carefully and make the necessary adjustment**. I think there are a couple of extensions that does that automatically; for example,` prettier in vscode.  `
 
<img width="515" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/223843193-8d2be208-b19b-4627-b015-a5cb6228f5a1.png">

Reply:
> Okay. Thanks for your feedback and advice. However, I saw a lot of `prettier extensions`. Do you mean prettier code formatter

Yes, prettier code formatter. You're right.